.github/workflows/build.yml

```diff
@@ -69,15 +68,18 @@ jobs:
         sed -i '/version: git/d' snapcraft.yaml
         yq -i 'del(.parts.cleanup)' snapcraft.yaml
       if: matrix.arch == 'aarch64'
-    - uses: diddlesnaps/snapcraft-multiarch-action@v1
-      if: matrix.arch == 'aarch64'
+    - uses: diddledani/snapcraft-multiarch-action@v1
+      #if: matrix.arch == 'aarch64'
+      if: ${{ false }} # needs some changes in the action to incorporate core24
       id: snapcraft_docker
       with:
         architecture: arm64
     - uses: snapcore/action-build@v1
       id: snapcraft
       if: matrix.arch == 'x86_64'
-    - uses: actions/upload-artifact@v3
+      env:
+        SNAPCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: 1
```

Comment:
I can't ship it with this, can I?

Reply:
Yes you can, but for now only amd64 and just from github actions.